rhscl-postgresql-apb/apb.yml

```diff
@@ -0,0 +1,31 @@
+id: 8a049637-8c70-4434-993c-a15c3d310513
+name: rhscl-postgresql-apb
+image: ansibleplaybookbundle/rhscl-postgresql-apb
+description: SCL PostgreSQL apb implementation
+bindable: true
+async: optional
+metadata:
+  displayName: "Red Hat SCL Postgresql"
+  longDescription: "An apb that deploys postgresql 9.4 or 9.5."
+  imageUrl: "https://wiki.postgresql.org/images/a/a4/PostgreSQL_logo.3colors.svg"
+  documentationUrl: "https://www.postgresql.org/docs/"
+parameters:
+  - name: namespace
```

Comment:
This goes beyond your PR, so not a blocker for merge or anything, but I wonder if we should make namespace a required parameter just to avoid the inevitable collision.